dist/api_data.py

```diff
@@ -574,6 +574,11 @@ def __ge__(self, other):
         Config('hashsize', '1024', r'''
             number of buckets in the hashtable that keeps track of objects''',
             min='64', max='1048576'),
+        Config('pinned', '', r'''
+            List of "table:" URIs exempt from cache eviction. Capacity config overrides this,
+            tables exceeding capacity will not be fully retained. Table names can appear
+            in both this and preload list, but not in both this and exclude list.''',
```

Comment:
A few nitpicks:
* "preload list" -> "the preload list"
* "exclude list" -> "the exclude list"
* We allow duplicate names, it might be worth mentioning that.

Reply:
Updated in [5e9ac89](https://github.com/wiredtiger/wiredtiger/pull/9415/commits/5e9ac89d77983d0f50b96a1d984098d292cc517a)